go.mod

```diff
@@ -13,10 +14,9 @@ require (
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/hashicorp/go-uuid v1.0.1
 	github.com/hashicorp/go-version v1.1.0
-	github.com/hashicorp/terraform v0.12.6
+	github.com/hashicorp/terraform v0.12.8
 	github.com/satori/go.uuid v1.2.0
 	github.com/satori/uuid v0.0.0-20160927100844-b061729afc07
-	github.com/sirupsen/logrus v1.2.0 // indirect
```

Comment:
since there's no changes to the vendored files, could we revert the changes to the `go.mod` and `go.sum`?

Reply:
Yes, thanks!  I had some module trouble initially trying to build the provider locally.